misc/sample_application/uservm/Makefile

```diff
@@ -1,12 +1,16 @@
-CC = gcc
-CP = g++
+CC ?= gcc
+CP ?= g++
```

Comment:
The typical name for g++ is `CXX`, not `CP`.

Reply:
> The typical name for g++ is `CXX`, not `CP`.

 Replace the "CP ?= g++" with "`CXX`?=g++" . thanks